### DISCUSSION TITLE
Embed libraries availability

### DISCUSSION THREAD

**Discussion by rmanhaeve** [pypa/manylinux#1724] [Q&A] [answered]
Hi

Everyone. I've been trying to build the [janus-swi package](https://github.com/swi-prolog/packages-swipy/), which allows bi-directional communication between Python and SWI-Prolog. 

However, there is an issue since this requires both the Development.Embed and Development.Module libraries.
I've read that manylinux only provides the latter. I assume there is a good reason for this, but it is not immediately clear to me. Is there an inherent reason why the embed library cannot be included?

Kind regards,
Robin

**Comment by mayeut** [comment, marked as answer]
The embed library are available but hidden for reasons outlined in #91 / #1250.

**Comment by rmanhaeve** [comment]
Thank you!
